cmd/spawn/module.go

```diff
@@ -17,11 +17,17 @@ import (
 	lang "golang.org/x/text/language"
 )
 
-const FlagIsIBCMiddleware = "ibc-middleware"
+const (
+	FlagIsIBCMiddleware = "ibc-middleware"
+)
+
+type features struct {
+	ibcMiddleware bool
```

Comment:
This is nice to have once we have more features to enable (I did for ORM originally as an opt in. then changed my mind as we should always have ORM for any module).